### PR TITLE
Polish Azure.ResourceManager.DevCenter for first GA

### PR DESCRIPTION
**Pull request by tbyfield**
Updating Azure.ResourceManager.DevCenter to published stable API version (2023-04-01)

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/devcenter/Azure.ResourceManager.DevCenter/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/CHANGELOG.md:3]: What's the expected release date?

File: sdk/devcenter/Azure.ResourceManager.DevCenter/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+- Corrected the format of all `ResouceType` type properties / parameters.
+- Corrected the format of all `AzureLocation` type properties / parameters.
+- Optimized the name of some models and functions.
+
 ### Bugs Fixed
 
 ### Other Changes
 
+- Upgraded dependent `Azure.Core` to `1.32.0`.
+- Upgraded dependent `Azure.ResourceManager` to `1.5.0`.
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/CHANGELOG.md:25]: 1.6.0

File: sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md [hunk 3]
```diff
@@ -9,21 +9,53 @@ generate-model-factory: false
 csharp: true
 library-name: DevCenter
 namespace: Azure.ResourceManager.DevCenter
-# default tag is a preview version
-require: https://github.com/Azure/azure-rest-api-specs/blob/1b3b9c1dd4d2c875997ea0b392dc71418fb1f28d/specification/devcenter/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/07c55de803057861912799405580ea9d022853fc/specification/devcenter/resource-manager/readme.md
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md:12]: Set the generate-model-factory: false to generate-model-factory: true which enables the model factory.

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 4]
```diff
@@ -141,48 +171,332 @@ public partial class DevCenterCollection : Azure.ResourceManager.ArmCollection,
...
+        public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation> DeleteAsync(Azure.WaitUntil waitUntil, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.DevCenter.DevCenterNetworkConnectionResource> Get(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.DevCenter.DevCenterNetworkConnectionResource>> GetAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.ResourceManager.DevCenter.HealthCheckStatusDetailResource GetHealthCheckStatusDetail() { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.DevCenter.Models.OutboundEnvironmentEndpoint> GetOutboundNetworkDependenciesEndpoints(int? top = default(int?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:379]: Rename this method to `GetOutboundEnvironmentEndpoints`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 5]
```diff
@@ -200,111 +514,71 @@ public partial class DevCenterResource : Azure.ResourceManager.ArmResource
...
-    public partial class GalleryData : Azure.ResourceManager.Models.ResourceData
+    public partial class DevCenterScheduleData : Azure.ResourceManager.Models.ResourceData
     {
-        public GalleryData() { }
-        public string GalleryResourceId { get { throw null; } set { } }
-        public string ProvisioningState { get { throw null; } }
+        public DevCenterScheduleData() { }
+        public Azure.ResourceManager.DevCenter.Models.ScheduledFrequency? Frequency { get { throw null; } set { } }
+        public Azure.ResourceManager.DevCenter.Models.DevCenterProvisioningState? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.DevCenter.Models.ScheduledType? ScheduledType { get { throw null; } set { } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:558]: `ScheduledType` is too generic

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 6]
```diff
@@ -200,111 +514,71 @@ public partial class DevCenterResource : Azure.ResourceManager.ArmResource
...
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
     }
-    public partial class GalleryData : Azure.ResourceManager.Models.ResourceData
+    public partial class DevCenterScheduleData : Azure.ResourceManager.Models.ResourceData
     {
-        public GalleryData() { }
-        public string GalleryResourceId { get { throw null; } set { } }
-        public string ProvisioningState { get { throw null; } }
+        public DevCenterScheduleData() { }
+        public Azure.ResourceManager.DevCenter.Models.ScheduledFrequency? Frequency { get { throw null; } set { } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:556]: `ScheduledFrequency` is too generic

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 7]
```diff
@@ -200,111 +514,71 @@ public partial class DevCenterResource : Azure.ResourceManager.ArmResource
...
+        public Azure.ResourceManager.DevCenter.Models.DevCenterProvisioningState? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.DevCenter.Models.ScheduledType? ScheduledType { get { throw null; } set { } }
+        public Azure.ResourceManager.DevCenter.Models.ScheduleEnableStatus? State { get { throw null; } set { } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:559]: `ScheduleEnableStatus` is too generic

File: sdk/devcenter/Azure.ResourceManager.DevCenter/CHANGELOG.md (hunk 1 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/CHANGELOG.md]: Set it to today.

File: sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md (hunk 3 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md:12]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 4 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 5 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Prepended `DevCenter`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 6 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Prepended `DevCenter`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 7 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Prepended `DevCenter`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 8]
```diff
@@ -372,7 +610,7 @@ public partial class ImageVersionData : Azure.ResourceManager.Models.ResourceDat
         public bool? ExcludeFromLatest { get { throw null; } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:610]: Is `DoesExcludeFromLatest` better?

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 9]
```diff
@@ -372,7 +610,7 @@ public partial class ImageVersionData : Azure.ResourceManager.Models.ResourceDat
...
         public string NamePropertiesName { get { throw null; } }
         public int? OSDiskImageSizeInGb { get { throw null; } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:612]: `Gb` -> `GB`, this should apply to all if there have others

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 8 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:610]: Maybe `IsExcluedFromLatest`?

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 10]
```diff
@@ -569,12 +687,12 @@ public partial class ProjectEnvironmentTypeCollection : Azure.ResourceManager.Ar
     public partial class ProjectEnvironmentTypeData : Azure.ResourceManager.Models.TrackedResourceData
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:687]: Is `DevCenterProjectEnvironment` better?

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 11]
```diff
@@ -141,48 +171,332 @@ public partial class DevCenterCollection : Azure.ResourceManager.ArmCollection,
...
+        public virtual Azure.Response<Azure.ResourceManager.DevCenter.DevCenterEnvironmentTypeResource> Update(Azure.ResourceManager.DevCenter.Models.DevCenterEnvironmentTypePatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.DevCenter.DevCenterEnvironmentTypeResource>> UpdateAsync(Azure.ResourceManager.DevCenter.Models.DevCenterEnvironmentTypePatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
     public static partial class DevCenterExtensions
     {
+        public static Azure.Response<Azure.ResourceManager.DevCenter.Models.CheckNameAvailabilityResponse> ExecuteDevCenterCheckNameAvailabilityAsync(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.ResourceManager.DevCenter.Models.CheckNameAvailabilityContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:221]: Please follow the review guide of `CheckNameAvailability` to change the method, response and request model names

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 12]
```diff
@@ -596,105 +714,289 @@ public partial class ProjectEnvironmentTypeResource : Azure.ResourceManager.ArmR
         public virtual Azure.Response<Azure.ResourceManager.DevCenter.ProjectEnvironmentTypeResource> Update(Azure.ResourceManager.DevCenter.Models.ProjectEnvironmentTypePatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.DevCenter.ProjectEnvironmentTypeResource>> UpdateAsync(Azure.ResourceManager.DevCenter.Models.ProjectEnvironmentTypePatch patch, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class ProjectResource : Azure.ResourceManager.ArmResource
+}
+namespace Azure.ResourceManager.DevCenter.Models
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CatalogSyncState : System.IEquatable<Azure.ResourceManager.DevCenter.Models.CatalogSyncState>
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:755]: consider to prepend rp name as prefix

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 13]
```diff
@@ -596,105 +714,289 @@ public partial class ProjectEnvironmentTypeResource : Azure.ResourceManager.ArmR
...
-        public string Time { get { throw null; } set { } }
-        public string TimeZone { get { throw null; } set { } }
-        public Azure.ResourceManager.DevCenter.Models.ScheduledType? TypePropertiesType { get { throw null; } set { } }
+        public CheckNameAvailabilityContent() { }
+        public string Name { get { throw null; } set { } }
+        public Azure.Core.ResourceType? ResourceType { get { throw null; } set { } }
     }
-    public partial class ScheduleResource : Azure.ResourceManager.ArmResource
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CheckNameAvailabilityReason : System.IEquatable<Azure.ResourceManager.DevCenter.Models.CheckNameAvailabilityReason>
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:781]: According to the review guide, this need rename to DevCenterNameUnavailableReason

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 14]
```diff
@@ -596,105 +714,289 @@ public partial class ProjectEnvironmentTypeResource : Azure.ResourceManager.ArmR
...
     }
-}
-namespace Azure.ResourceManager.DevCenter.Models
-{
-    public partial class Capability
+    public partial class CheckNameAvailabilityResponse
+    {
+        internal CheckNameAvailabilityResponse() { }
+        public string Message { get { throw null; } }
+        public bool? NameAvailable { get { throw null; } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:802]: `IsNameAvailable`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 15]
```diff
@@ -596,105 +714,289 @@ public partial class ProjectEnvironmentTypeResource : Azure.ResourceManager.ArmR
...
+        public string DomainPassword { get { throw null; } set { } }
+        public string DomainUsername { get { throw null; } set { } }
+        public string OrganizationUnit { get { throw null; } set { } }
+        public Azure.Core.ResourceIdentifier SubnetId { get { throw null; } set { } }
+    }
+    public partial class DevCenterOperationStatus : Azure.ResourceManager.Models.OperationStatusResult
+    {
+        internal DevCenterOperationStatus() : base (default(string)) { }
+        public System.BinaryData Properties { get { throw null; } }
+        public string ResourceId { get { throw null; } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:964]: Is this a `arm-id`?

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 16]
```diff
@@ -718,111 +1020,91 @@ public enum DevCenterSkuTier
...
+    public readonly partial struct DomainJoinType : System.IEquatable<Azure.ResourceManager.DevCenter.Models.DomainJoinType>
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public EnableStatus(string value) { throw null; }
-        public static Azure.ResourceManager.DevCenter.Models.EnableStatus Disabled { get { throw null; } }
-        public static Azure.ResourceManager.DevCenter.Models.EnableStatus Enabled { get { throw null; } }
-        public bool Equals(Azure.ResourceManager.DevCenter.Models.EnableStatus other) { throw null; }
+        public DomainJoinType(string value) { throw null; }
+        public static Azure.ResourceManager.DevCenter.Models.DomainJoinType AzureADJoin { get { throw null; } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:1157]: Rename to `AadJoin`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 17]
```diff
@@ -718,111 +1020,91 @@ public enum DevCenterSkuTier
...
+        public static Azure.ResourceManager.DevCenter.Models.DomainJoinType HybridAzureADJoin { get { throw null; } }
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:1158]: Rename to `HybridAadJoin`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs [hunk 18]
```diff
@@ -718,111 +1020,91 @@ public enum DevCenterSkuTier
...
         public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.ResourceManager.DevCenter.Models.DomainJoinType left, Azure.ResourceManager.DevCenter.Models.DomainJoinType right) { throw null; }
-        public static implicit operator Azure.ResourceManager.DevCenter.Models.DomainJoinType (string value) { throw null; }
-        public static bool operator !=(Azure.ResourceManager.DevCenter.Models.DomainJoinType left, Azure.ResourceManager.DevCenter.Models.DomainJoinType right) { throw null; }
+        public static bool operator ==(Azure.ResourceManager.DevCenter.Models.DevCenterUsageUnit left, Azure.ResourceManager.DevCenter.Models.DevCenterUsageUnit right) { throw null; }
+        public static implicit operator Azure.ResourceManager.DevCenter.Models.DevCenterUsageUnit (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.DevCenter.Models.DevCenterUsageUnit left, Azure.ResourceManager.DevCenter.Models.DevCenterUsageUnit right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class DevCenterUserRoleAssignmentValue
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:1146]: `DevCenterUserRoleAssignments` is better

Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:1146]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 17 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:1158]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 16 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:1157]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 15 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs:964]: Yes, the config in autorest.md was commented out accidentally, fixed now.

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 14 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 13 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 12 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 10 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs (hunk 9 above)
Comment (live1206) [reply sdk/devcenter/Azure.ResourceManager.DevCenter/api/Azure.ResourceManager.DevCenter.netstandard2.0.cs]: Updated

File: sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md [hunk 19]
```diff
@@ -51,6 +86,46 @@ rename-rules:
...
+  ScheduleEnableStatus.Enabled: IsEnabled
+  ScheduleEnableStatus.Disabled: IsDisabled
+  StopOnDisconnectEnableStatus.Enabled: IsEnabled
+  StopOnDisconnectEnableStatus.Disabled: IsDisabled
+  AttachedNetworkConnection.properties.networkConnectionLocation: -|azure-location
+  UserRoleAssignmentValue: DevCenterUserRoleAssignments
+  DomainJoinType.HybridAzureADJoin: HybridAadJoin
+  DomainJoinType.AzureADJoin: AadJoin
+  CheckNameAvailabilityResponse.nameAvailable: IsNameAvailable
+  CheckNameAvailabilityRequest: CheckDevCenterNameAvailabilityContent
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md:116]: Remove the `Check` prefix

File: sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md [hunk 20]
```diff
@@ -51,6 +86,46 @@ rename-rules:
...
+  CheckNameAvailabilityResponse: CheckDevCenterNameAvailabilityResult
+  CheckNameAvailabilityReason: CheckDevCenterUnavailableReason
+  ProjectEnvironmentType: DevCenterProjectEnvironment
+  ImageVersion.properties.osDiskImageSizeInGb:  OsDiskImageSizeInGB
+  ImageVersion.properties.excludeFromLatest: IsExcludedFromLatest
+
+override-operation-name:
+  OperationStatuses_Get: GetDevCenterOperationStatus
+  Usages_ListByLocation: GetDevCenterUsagesByLocationAsync
+  CheckNameAvailability_Execute: ExecuteCheckDevCenterNameAvailabilityAsync
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md:126]: Rename to `CheckDevCenterNameAvailability`

File: sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md [hunk 21]
```diff
@@ -51,6 +86,46 @@ rename-rules:
...
+  Skus_ListBySubscription: GetDevCenterSkusBySubscriptionAsync
```
Comment (ArthurMa1978) [inline sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md:127]: Dont need the Async suffix

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**